src/com/amazon/ion/impl/lite/IonValueLite.java

```diff
@@ -633,7 +633,7 @@ public SymbolTable getSymbolTable()
         if (symbols != null) {
             return symbols;
         }
-        return _context.getSystem().getSystemSymbolTable();
+        return getSystem().getSystemSymbolTable();
```

Comment:
`getSystem()` is already implemented using `_context.getSystem()` inside `IonValueLite`, so this was redundant.